src/patternfly/components/Tooltip/tooltip.scss

```diff
@@ -68,8 +66,8 @@
 .pf-c-tooltip__arrow {
   position: absolute;
   width: var(--pf-c-tooltip__arrow--Width);
-  height: var(--pf-c-tooltip__arrow--Height);
```

Comment:
Do you intent to assign the --Width variable to the height? If so, I get why, but at the least it deserves a comment explaining, so it doesn't look like a mistake.

Reply:
added a comment.